### PR TITLE
Improve error messages when global paths are required

### PR DESCRIPTION
**Pull request by McMartin**
Related to my comment there: https://github.com/McMartin/FRUT/pull/638#pullrequestreview-423895627

@importjingles, @MartyLake: please review, thanks!

### CODE REVIEW

File: Jucer2Reprojucer/main.cpp [hunk 1]
```diff
@@ -1366,12 +1370,12 @@ int main(int argc, char* argv[])
       wLn("  PATH \"",
           useGlobalPath ? (isJuceModule ? "${JUCE_MODULES_GLOBAL_PATH}"
                                         : "${USER_MODULES_GLOBAL_PATH}")
-                        : relativeModulePath.replace("\\", "/"),
+                        : modulePath.replace("\\", "/"), //
```
Comment (importjingles) [inline Jucer2Reprojucer/main.cpp:1373]: Is the comment superfluous? Otherwise looks good :)

Comment (McMartin) [reply Jucer2Reprojucer/main.cpp:1373]: Very good catch! I added that comment to track down the origin of the `relativeModulePath` variable using `gitk`'s "Origin of this line" feature. I simply forgot to remove it before making the commit :sweat_smile:

Comment (McMartin) [reply Jucer2Reprojucer/main.cpp]: Addressed in 0547d00b8762d96ec83194ccc5af1e3305279994.

**Comment by McMartin** [comment]
@importjingles Please have a look at the last commit and let me know whether I should change anything. Thanks!

**Pull request merged**